src/status_im2/common/scroll_page/view.cljs

```diff
@@ -0,0 +1,76 @@
+(ns status-im2.common.scroll-page.view
+  (:require
+   [react-native.core :as rn]
+   [quo2.core :as quo]
+   [utils.re-frame :as rf]
+   [quo2.foundations.colors :as colors]
+   [status-im2.common.scroll-page.style :as style]
+   [oops.core :as oops]
+   [reagent.core :as reagent]
+   [quo.platform :as platform]))
+
+(defn icon-color []
+  (colors/theme-colors
+   colors/white-opa-40
+   colors/neutral-80-opa-40))
+
+(defn get-platform-value [value] (if platform/ios? (+ value 44) value))
```

Comment:
initially I was going to re use this get platform value function. 

I realise now it's only used in one place so I just removed that function and put it inline.

Yeah, it could be nice to encapsulate the scroll2 etc values. This is all done with callbacks so I'll consider passing them to the function via a render prop